### PR TITLE
refactor(query): replacing object.get with valid_key function for Terraform

### PR DESCRIPTION
**Pull request by tiagocruz20**
**Proposed Changes**
- refactoring object.get function to valid_key function

I submit this contribution under the Apache-2.0 license.

### CODE REVIEW

File: assets/queries/terraform/aws/db_instance_storage_not_encrypted/query.rego [hunk 1]
```diff
@@ -1,12 +1,14 @@
 package Cx
 
+import data.generic.common as common_lib
+
 CxPolicy[result] {
 	resource := input.document[i].resource.aws_db_instance[name]
 
-	object.get(resource, "storage_encrypted", "undefined") != "undefined"
+	common_lib.valid_key(resource, "storage_encrypted")
 	not resource.storage_encrypted
```
Comment (rafaela-soares) [inline assets/queries/terraform/aws/db_instance_storage_not_encrypted/query.rego:9]: `resource.storage_encrypted == false`

**Review by rafaela-soares — CHANGES_REQUESTED** [review]
Good job, @tiagocruz20!  

Overall, LGTM. I suggest the following small changes:

File: assets/queries/terraform/aws/elasticsearch_logs_disabled/query.rego [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
 
-import data.generic.common as commonLib
+import data.generic.common as common_lib
 
 CxPolicy[result] {
 	awsElasticsearchDomain := input.document[i].resource.aws_elasticsearch_domain[name]
-	object.get(awsElasticsearchDomain, "log_publishing_options", "undefined") == "undefined"
+	not common_lib.valid_key(awsElasticsearchDomain, "log_publishing_options")
 
 	result := {
```
Comment (rafaela-soares) [inline assets/queries/terraform/aws/elasticsearch_logs_disabled/query.rego:9]:
```
"keyExpectedValue": "'log_publishing_options' is defined and not null",
"keyActualValue": "'log_publishing_options' is undefined or null",
```

**Review by rafaela-soares — CHANGES_REQUESTED** [review]

**Review by rogeriopeixotocx — APPROVED** [review]
LGTM

**Review by rafaela-soares — APPROVED** [review]

**Pull request merged**